### PR TITLE
 mod(es/ast): remove useless fields in private prop 

### PR DESCRIPTION
**Pull request by Austaras**
**Description:**

remove `abstract` `declare` and `computed`

**BREAKING CHANGE:**
It will be a break change for rust users

**Related issue (if exists):**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+        "**/.DS_Store": true,
+        "**/*.bk": true,
+        "**/node_modules": true,
+        "**/target": true
+    },
+    "[rust]": {
+        "editor.formatOnSave": true
+    },
+    "[typescript]": {
+        // "editor.formatOnSave": true
```
Comment (kdy1) [inline .vscode/settings.json:16]: Is this intentional?

Comment (kdy1) [reply .vscode/settings.json:16]: Hmm... I'm fine with it.

**Review by kdy1 — DISMISSED** [review]
Thanks!

---

swc-bump:
 - swc_ecma_ast --breaking

File: .vscode/settings.json (hunk 1 above)
Comment (Austaras) [reply .vscode/settings.json]: No

**Review by kdy1 — APPROVED** [review]

**Review by kdy1 — CHANGES_REQUESTED** [review]
Oh, then please revert it.

**Comment by Austaras** [comment]
Sorry for pushing unrelated changes.

**Review by kdy1 — DISMISSED** [review]
It's fine, thank you anyway!

**Comment by kdy1** [comment]
CI failed due to typescript errors

**Comment by Austaras** [comment]
What do node-swc `visitor.ts` do? For JS plugins?

**Comment by kdy1** [comment]
Yes

**Review by kdy1 — APPROVED** [review]

**Pull request merged**